### ISSUE TITLE
False positive for "export const { a, b } = c"

### ISSUE THREAD

**Issue by emlai** [nadeesha/ts-prune#15]
ts-prune reports false positives for the following exports:

```ts
import * as aws from './aws'
export const { getFileUrl, saveToStorage } = aws
```

They're imported like this:

```ts
import * as storage from '../s3/storage'
// used as `storage.saveToStorage`
```

`storage.getFileUrl` is also imported and used the same way in a different file.

Error message:

```
{ getFileUrl, saveToStorage } @ /path/to/storage.ts:2
```

If I export them as follows then the false positives are gone:

```ts
export const getFileUrl = aws.getFileUrl
export const saveToStorage = aws.saveToStorage
```

**Comment by nadeesha** [comment]
Thank you this looks legitimate. Adding to the backlog.

**Comment by nadeesha** [comment]
Fixed by #41.
